source/chapters/microphones.rst

```diff
@@ -31,13 +34,7 @@ Software monitoring can be used with the soundcard build into a computer, but
 it has a big drawback. Due to the nature of digital audio, it takes time for
 input from the soundcard to be available for Mixxx to process, time for Mixxx
 to process the audio, and more time to send the audio back out the soundcard.
-This time is referred to as "latency". Mixxx can be configured to run at low
-latencies by choosing a smaller :ref:`preferences-audio-buffer`, but there is no
-way to completely eliminate latency. Thus, if you use software monitoring, you
-will hear the microphone signal out of the speakers after you make sound into
-the microphone. Although latency is measured in milliseconds, even a few
-milliseconds can be disorienting and distracting, whether you are using the
-microphone for spoken announcements, vocals, or playing a musical instrument.
+This time is referred to as "latency".
 
 .. figure:: ../_static/software-monitor-signal-path.png
```

Comment:
I am missing this file, `manual/build/html/chapters/_static/software-monitor-signal-path.png` along with some other warnings for pngs:

```
/home/gramanas/Code/manual/source/chapters/hardware.rst:: WARNING: image file not readable: chapters/../_static/mixxx_setup_splitter_adaptors.png
/home/gramanas/Code/manual/source/chapters/microphones.rst:9: WARNING: Unexpected indentation.
/home/gramanas/Code/manual/source/chapters/microphones.rst:46: WARNING: image file not readable: chapters/../_static/software-monitor-signal-path.png
/home/gramanas/Code/manual/source/chapters/microphones.rst:80: WARNING: image file not readable: chapters/../_static/direct-monitor-signal-path.png
/home/gramanas/Code/manual/source/chapters/microphones.rst:121: WARNING: image file not readable: chapters/../_static/direct-monitor-loopback-signal-path.png
/home/gramanas/Code/manual/source/chapters/microphones.rst:150: WARNING: image file not readable: chapters/../_static/direct-monitor-input-latency.png
/home/gramanas/Code/manual/source/chapters/microphones.rst:187: WARNING: image file not readable: chapters/../_static/latency-measurement.png
/home/gramanas/Code/manual/source/chapters/microphones.rst:246: WARNING: image file not readable: chapters/../_static/external-mixing-with-microphone-signal-path.png
/home/gramanas/Code/manual/source/chapters/microphones.rst:266: WARNING: image file not readable: chapters/../_static/external-mixing-with-microphone-signal-path.png
/home/gramanas/Code/manual/source/chapters/microphones.rst:316: WARNING: image file not readable: chapters/../_static/dj-controller-with-mic-and-splitter-cables.png
/home/gramanas/Code/manual/source/chapters/quickstart.rst:4: WARNING: Unknown target name: "table of contents".
/home/gramanas/Code/manual/source/chapters/user_interface.rst:106: WARNING: image file not readable: chapters/../_static/Mixxx-200-Latenight-Deck-Waveform.png
```

I am not familiar with rst so I might be doing something wrong, in that case discard the comment!

Reply:
Those diagrams have not been made yet. One step at a time :)

Reply:
Ah, fair enough!